specifications/api/swagger.yaml

```diff
@@ -1285,3 +1291,27 @@ paths:
         - <<: *parametersBody
           schema: *ApiMigrateByronWalletData
       responses: *responsesMigrateByronWallet
+
+  /network/information:
+    get:
+        operationId: getNetworkInformation
+        tags: ["Network"]
+        summary: Information
+        description: |
+          <p align="right">status: <strong>stable</strong></p>
+        responses: *responsesGetNetworkInformation
+
+  /proxy/transactions:
```

Comment:
I'm feeling lukewarm about the name "proxy" (conveys mechanism), but this is probably splitting hairs.